node/src/database/db_migrations.rs

```diff
@@ -507,50 +511,54 @@ mod tests {
         let iterator = list_of_updates.iter();
         let iterator_shifted = list_of_updates.iter().skip(1);
         iterator.zip(iterator_shifted).for_each(|(first, second)| {
-            if compare_two_numbers(first.old_version(), second.old_version()).not() {
+            if assert_on_two_numbers_tight_continuity(first.old_version(), second.old_version())
+                .not()
+            {
                 panic!("The list of updates for the database is not ordered properly")
             }
         });
     }
 
-    fn compare_two_numbers(first: usize, second: usize) -> bool {
-        first.le(&second)
+    fn assert_on_two_numbers_tight_continuity(first: usize, second: usize) -> bool {
+        (first + 1) == second
     }
 
     #[test]
-    fn list_of_existing_updates_is_correctly_ordered() {
-        let _ = list_validation_check(DbMigratorReal::list_of_existing_updates());
+    fn list_of_approved_updates_is_correctly_ordered() {
+        let _ = list_validation_check(DbMigratorReal::list_of_approved_updates());
         //success if no panicking
     }
 
     #[test]
-    fn list_of_existing_updates_does_not_end_with_version_higher_than_the_current_version() {
-        let last_entry = DbMigratorReal::list_of_existing_updates()
+    fn list_of_approved_updates_does_not_end_with_version_higher_than_the_current_version() {
```

Comment:
The test is stronger than this name implies. How about ...`ends_on_the_current_version`?